Cannot write to file '/build/outputs/dexcount/debugChart' specified for property 'chartDir' as it is a directory.

## Command:

`./gradlew clean assembleDebug countDebugDexMethods`

## Error:
```
:assembleDebug
Total methods in -debug.aar: 2154 (3.29% used)
Total fields in -debug.aar:  1036 (1.58% used)
Methods remaining in -debug.aar: 63381
Fields remaining in -debug.aar:  64499
:countDebugDexMethods FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':countDebugDexMethods'.
> Cannot write to file '/build/outputs/dexcount/debugChart' specified for property 'chartDir' as it is a directory.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 9.439 secs
```

gradle --stacktrace countDebugDexMethods
	Incremental java compilation is an incubating feature.
	:app:countDebugDexMethods FAILED

	FAILURE: Build failed with an exception.

	* What went wrong:
	A problem was found with the configuration of task ':app:countDebugDexMethods'.
	> Cannot write to file '/Users/charliehubbard/projects/android/AmplifiedNextGen/app/build/outputs/dexcount/debugChart' specified for property 'chartDir' as it is a directory.

	* Try:
	Run with --info or --debug option to get more log output.

	* Exception is:
	org.gradle.api.tasks.TaskValidationException: A problem was found with the configuration of task ':app:countDebugDexMethods'.
		at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:55)
		at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:52)
		at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
		at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:53)
		at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
		at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:203)
		at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:185)
		at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.processTask(AbstractTaskPlanExecutor.java:66)
		at org.gradle.execution.taskgraph.AbstractTaskPlanExecutor$TaskExecutorWorker.run(AbstractTaskPlanExecutor.java:50)
		at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor.process(DefaultTaskPlanExecutor.java:25)
		at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter.execute(DefaultTaskGraphExecuter.java:110)
		at org.gradle.execution.SelectedTaskExecutionAction.execute(SelectedTaskExecutionAction.java:37)
		at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.java:37)
		at org.gradle.execution.DefaultBuildExecuter.access$000(DefaultBuildExecuter.java:23)
		at org.gradle.execution.DefaultBuildExecuter$1.proceed(DefaultBuildExecuter.java:43)
		at org.gradle.execution.DryRunBuildExecutionAction.execute(DryRunBuildExecutionAction.java:32)
		at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.java:37)
		at org.gradle.execution.DefaultBuildExecuter.execute(DefaultBuildExecuter.java:30)
		at org.gradle.initialization.DefaultGradleLauncher$4.run(DefaultGradleLauncher.java:154)
		at org.gradle.internal.Factories$1.create(Factories.java:22)
		at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
		at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:52)
		at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:151)
		at org.gradle.initialization.DefaultGradleLauncher.access$200(DefaultGradleLauncher.java:32)
		at org.gradle.initialization.DefaultGradleLauncher$1.create(DefaultGradleLauncher.java:99)
		at org.gradle.initialization.DefaultGradleLauncher$1.create(DefaultGradleLauncher.java:93)
		at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
		at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:62)
		at org.gradle.initialization.DefaultGradleLauncher.doBuild(DefaultGradleLauncher.java:93)
		at org.gradle.initialization.DefaultGradleLauncher.run(DefaultGradleLauncher.java:82)
		at org.gradle.launcher.exec.InProcessBuildActionExecuter$DefaultBuildController.run(InProcessBuildActionExecuter.java:94)
		at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:28)
		at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
		at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:43)
		at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:28)
		at org.gradle.launcher.exec.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:75)
		at org.gradle.launcher.exec.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:45)
		at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.execute(DaemonUsageSuggestingBuildActionExecuter.java:51)
		at org.gradle.launcher.exec.DaemonUsageSuggestingBuildActionExecuter.execute(DaemonUsageSuggestingBuildActionExecuter.java:28)
		at org.gradle.launcher.cli.RunBuildAction.run(RunBuildAction.java:43)
		at org.gradle.internal.Actions$RunnableActionAdapter.execute(Actions.java:170)
		at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:237)
		at org.gradle.launcher.cli.CommandLineActionFactory$ParseAndBuildAction.execute(CommandLineActionFactory.java:210)
		at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:35)
		at org.gradle.launcher.cli.JavaRuntimeValidationAction.execute(JavaRuntimeValidationAction.java:24)
		at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:206)
		at org.gradle.launcher.cli.CommandLineActionFactory$WithLogging.execute(CommandLineActionFactory.java:169)
		at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:33)
		at org.gradle.launcher.cli.ExceptionReportingAction.execute(ExceptionReportingAction.java:22)
		at org.gradle.launcher.Main.doAction(Main.java:33)
		at org.gradle.launcher.bootstrap.EntryPoint.run(EntryPoint.java:45)
		at org.gradle.launcher.bootstrap.ProcessBootstrap.runNoExit(ProcessBootstrap.java:54)
		at org.gradle.launcher.bootstrap.ProcessBootstrap.run(ProcessBootstrap.java:35)
		at org.gradle.launcher.GradleMain.main(GradleMain.java:23)
	Caused by: org.gradle.api.InvalidUserDataException: Cannot write to file '/Users/charliehubbard/projects/android/AmplifiedNextGen/app/build/outputs/dexcount/debugChart' specified for property 'chartDir' as it is a directory.
		at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:47)
		... 53 more


	BUILD FAILED

	Total time: 8.142 secs

--- a/src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountTask.groovy
+++ b/src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountTask.groovy
@@ -25,6 +25,7 @@ import groovy.transform.stc.ClosureParams
 import groovy.transform.stc.SimpleType
 import org.gradle.api.DefaultTask
 import org.gradle.api.logging.LogLevel
+import org.gradle.api.tasks.OutputDirectory
 import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.TaskAction
 import org.gradle.logging.StyledTextOutput
@@ -50,7 +51,7 @@ class DexMethodCountTask extends DefaultTask {
     @OutputFile
     def File summaryFile
 
-    @OutputFile
+    @OutputDirectory
     def File chartDir
 
     def DexMethodCountExtension config

